.github/workflows/upload-translations.yml

```diff
@@ -0,0 +1,40 @@
+name: Update translations on the main branch
+
+on:
+  push:
+    paths:
+      - 'doc/**/*.rst'
+      - 'doc/locale/**/*.po'
```

Comment:
Why do I need to run a pull when this file changes?

Reply:
We merge translation files from a PR and then re-upload translations back to Crowdin